etc/httpd/global.conf

```diff
@@ -951,3 +951,6 @@ RewriteRule ^/(\w{2,3}(?:-\w{2})?/)?firefox/pocket(/?)$ /b/$1firefox/pocket$2 [P
 
 # bug 1136436
 RewriteRule ^/(\w{2,3}(?:-\w{2})?/)?firefox/ios(/?)$ /b/$1firefox/ios$2 [PT]
+
+# bug 1205632
+RewriteRule ^(/js/language(/.+)?)$ /b$1 [PT]
```

Comment:
One more thing. I think that this won't match `/js/language/' because the `.+` means it needs at least one character after the trailing slash. Just make it `.*` and we should be good.